_data/projects/leaflet.yml

```diff
@@ -0,0 +1,10 @@
+name: Leaflet
+desc: JavaScript library for mobile-friendly interactive maps
+site: http://leafletjs.com
+tags:
+- Javascript
+- Maps
+- GIS
+upforgrabs:
+  name: easy-fix
+  link: https://github.com/Leaflet/Leaflet/issues?q=is%3Aopen+label%3A%22easy+fix%22
```

Comment:
A bit of :lipstick: - this can be swapped out for https://github.com/Leaflet/Leaflet/labels/easy%20fix